recordedit/recordEdit.app.js

```diff
@@ -219,7 +219,9 @@
                                             break;
                                     }
 
-                                    if (!context.queryParams.copy || !column.getInputDisabled(context.appContext)) {
+                                    // line 181 checks for input disabled (generated) and not copy
+                                    // no need to check for copy here because that case above guards against the negative case for copy
+                                    if (!column.getInputDisabled(context.appContext)) {
```

Comment:
While this would work; it's a bit difficult to follow the code. You can change it to:
```javascript

// If input is disabled, there's no need to transform the column value.
if (column.getInputDisabled(context.appContext)) {

  // in copy mode, don't copy the disabled 
  if (context.mode != context.modes.COPY) {
    recordEditModel.rows[j][column.name] = values[i];
  }

 continue;
}

// Transform column values for use in view model
..

// this last if statement is not needed anymore
recordEditModel.rows[j][column.name] = value;
```
You can also break the first `if` statement into two different statements `if` you want.

```javascript
// Don't copy the disabled value
if (context.mode == context.modes.COPY && column.getInputDisabled(context.appContext)) {
  continue;
}

// Don't transform the disabled input
if (column.getInputDisabled(context.appContext)) {
  recordEditModel.rows[j][column.name] = values[i];
  continue;
}
```
I'm not sure which one is more readable.